Sources/tart/VMDirectory.swift

```diff
@@ -59,6 +59,19 @@ struct VMDirectory: Prunable {
     return VMDirectory(baseURL: tmpDir)
   }
 
+  static func temporaryDeterministic(fileName: String) throws -> VMDirectory {
+    let fileNameNoSlash = fileName.replacingOccurrences(of: "/", with: "-")
+    let tmpDir = try Config().tartTmpDir.appendingPathComponent(fileNameNoSlash)
+    do {
+      try FileManager.default.createDirectory(at: tmpDir, withIntermediateDirectories: false)
```

Comment:
I think you can pass `true` to `withIntermediateDirectories` and don't catch the exception.